api/Element.json

```
@@ -389,6 +389,57 @@
           }
         }
       },
+      "attributeStyleMap": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/attributes",
```

A little fix here please

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/attributeStyleMap",
```